### PR TITLE
Update API naming convention for entity/node/dcid

### PR DESCRIPTION
**Pull request by shifucun**
"node" represents a knowledge graph node which could be of any kind while "entity" should only be used for real world entities. This PR updates the API parameters and code usage to follow this pattern:

* "node" is used for node APIs: properties, property values and triples
* "entity" is used for observation APIs and it's mostly used together with "variable". This is current behavior and not changed in this PR.
* "dcid" is used for specialized API like /info/* where the actual entity type is already clear in the API name, so using "dcid" is better than "entity"

All these changes only affect website clients, which will be updated in one PR with mixer submodule update.

### CODE REVIEW

File: internal/server/v1/info/golden/bulk_place_info/bulk_place_info.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "data": [
     {
-      "entity": "earth"
+      "dcid": "earth"
     },
     {
-      "entity": "country/USA",
+      "dcid": "country/USA",
```
Comment (beets) [inline internal/server/v1/info/golden/bulk_place_info/bulk_place_info.json:7]: +1 to this change -- this brings more consistency to the response.

**Review by beets — COMMENTED** [review]
I haven't finished looking through everything, but wonder if the distinction between node and dcid is clear (especially in the API responses). Should we standardize on one or the other?

Also, do we know which API is used by RFF -- we should keep the change backwards compatible.

**Comment by pradh** [comment]
> That's a good point, it is indeed subtle to have both dcid and node. However like property value API, if we have both "property" and "dcid" in the request argument, would the "dcid" be too general in this case, since the property is also a dcid... Current treatment is to use "dcid" when the API is very specific about an entity like /info/place or /internal/bio

Why not use `entity` or `node` there too? In other words, I wonder about using dcid anywhere at all.

**Comment by shifucun** [comment]
Per comments, change "dcid" to "node" and we can take a look at how it works now

**Review by beets — APPROVED** [review]
+1 to node everywhere.

**Review by pradh — APPROVED** [review]

File: proto/v1/variable.proto [hunk 2]
```diff
@@ -16,11 +16,9 @@ syntax = "proto3";
 option go_package = "./proto";
 package datacommons.v1;
 
-import "stat_var.proto";
-
 message VariableAncestorsRequest {
   // DCID of a stat var (group)
-  string entity = 1;
+  string dcid = 1;
```
[the remark this answers is not in the document]
Comment (pradh) [reply proto/v1/variable.proto]: This is stale comment I lined up yesterday but didn't send.

File: proto/v1/page.proto [hunk 3]
```diff
@@ -17,11 +17,11 @@ option go_package = "./proto";
 package datacommons.v1;
 
 message BioPageRequest {
-  string entity = 1;
+  string dcid = 1;
 }
 
 message PlacePageRequest {
-  string entity = 1;
+  string dcid = 1;
```
[the remark this answers is not in the document]
Comment (pradh) [reply proto/v1/page.proto]: This is stale comment I lined up yesterday but didn't send.

File: proto/v1/triples.proto [hunk 4]
```diff
@@ -31,14 +37,14 @@ message TriplesRequest {
...
-  map<string, datacommons.EntityInfoCollection> triples = 1;
+  map<string, NodeInfoCollection> triples = 1;
   // The pagination token for getting the next set of entries.
   string next_token = 2;
 }
 
 
 message BulkTriplesRequest {
-  repeated string entities = 1;
+  repeated string nodes = 1;
```
[the remark this answers is not in the document]
Comment (shifucun) [reply proto/v1/triples.proto:47]: Yes, import browser uses the property values API. Will need to update there

**Pull request merged**